vuepress/docs/v7.0/docs/reference/deployment-structure.md

```diff
@@ -4,154 +4,128 @@ sidebarDepth: 2
-
-## app-builder
-The _app-builder_ is the front end of the _entando-de-app_. It communicates with the _entando-de-app_ via [REST 
-APIs](../consume/entando-apis.md). The _app-builder_ is a React JS application and is served via node in the default 
-deployment. In a quickstart deployment, the _app-builder_ container is deployed in the _entando-app_ 
-multiple container pod. The _app-builder_ also communicates with the Component Manager via REST API to fetch 
-information about Entando bundles deployed to the Entando Component Repository (ECR).
+The **entando-de-app** is very likely to be customized as part of an Entando implementation. 
+A customized image can include:
+* new APIs 
+* legacy Entando plugins
+* new database tables  
+* other extensions to the **entando-core**. 
+
+It is highly recommended that most extensions to the Entando platform occur in microservices. However, legacy 
+integrations, extensions to the CMS, and migrations from earlier Entando versions may require changes to the **entando-de-app**. 
+
+## App Builder
+The App Builder is the user-friendly front end UI for the **entando-de-app**. A ReactJS application, it is served via node in the default deployment. In a quickstart deployment, the App Builder container is deployed in the **entando-app** multi-container pod. It communicates with the **entando-de-app** and the Entando Component Manager (ECM) via [REST 
+APIs](../consume/entando-apis.md).  The ECM provides the information about bundles deployed to the Entando Component Repository (ECR).
 
 * GitHub: <https://github.com/entando/app-builder/>
 * DockerHub: <https://hub.docker.com/repository/docker/entando/app-builder/>
 
 #### Customization
-The _app-builder_ is customized as part of many Entando implementations. 
+The App Builder is customized as part of many Entando implementations. 
 It can be customized at runtime via micro frontends 
 [widget configuration](../../tutorials/create/mfe/widget-configuration.md).
 
-## component-manager(cm)
-The _component-manager_ provides the link between the entando-de-app (or your custom core instance) and the ECR. The _component-manager_ queries the entando-k8s service to fetch available 
-bundles that have been deployed as custom resources inside of an Entando cluster. 
-The _component-manager_ also manages the relationships between an Entando application and the 
-installed plugins. This can be seen in the plugin link custom resources in Kubernetes. 
+## Entando Component Manager (ECM)
+ECM provides the link between the **entando-de-app**, or your custom core instance, and the ECR. ECM queries the Entando Kubernetes service to fetch available 
+bundles that have been deployed as custom resources inside the cluster. 
+
+ECM also manages the relationships between an Entando application and the 
+installed plugins. This can be seen in the plugin link custom resource in Kubernetes. 
```

Comment:
awkward sentence on line 77.  what is seen where? are we talking about a link in K8s documentation? is the link called "custom resource"?? so confused